examples/main/package.json

```diff
@@ -13,11 +13,31 @@
     "immutable": "^3.8.1",
     "react": "^15.4.1",
     "react-dom": "^15.4.1",
-    "react-map-gl": ">=3.0.0-alpha.13"
+    "react-map-gl": ">=3.0.0-alpha.14"
   },
   "devDependencies": {
-    "buble-loader": "^0.4.0",
+    "autoprefixer-loader": "^3.2.0",
+    "babel-core": "^6.21.0",
+    "babel-loader": "^6.2.10",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-preset-es2015": "^6.18.0",
+    "babel-preset-react": "^6.16.0",
+    "babel-preset-stage-0": "^6.18.0",
```

Comment:
What features in `stage-0` are we using? It's a nit, but personally I wouldn't recommend using `stage-0`. 

`stage-2` represents mature features and includes the `class members` which I assume is the main thing we use to avoid decorators.